.github/workflows/track_todo.yml

```diff
@@ -0,0 +1,17 @@
+name: Track TODOs
+on:
+  push:
+    branches:
+      - develop
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Run tdg-github-action
+      uses: ribtoks/tdg-github-action@master
```

Comment:
Can we use a commit version of `ribtoks/tdg-github-action`  instead of master branch. Latest commit is `b389a3667310bc373cb771fe05dac180ce0351e6`